### ISSUE TITLE
Allow for 1000 satoshi fees.

### ISSUE THREAD

**Issue by dabura667** [bitpay/copay#2543]
10000 satoshi per kB flat fee is non-optimal.

There needs to be a more nuanced approach to the fee system in Copay.

However, for the interim, I would suggest having an advanced option in the settings which can set the fee from "Normal" to "Economy" similar to how Mycelium does it...

Maybe even go so far as to add advanced option when creating a new wallet that says "this wallet allows economy fees" or not.

tl;dr
We should somehow allow for 1000 satoshi per kB... or at least sub-10000 satoshi per kB fees.

**Comment by matiu** [comment]
agree. Note that currently is 10,000 satoshis per kB.

On Sun, Apr 19, 2015 at 12:13 AM, dabura667 <notifications@github.com>
wrote:

> 10000 satoshi flat fee is non-optimal.
>
> There needs to be a more nuanced approach to the fee system in Copay.
>
> However, for the interim, I would suggest having an advanced option in the
> settings which can set the fee from "Normal" to "Economy" similar to how
> Mycelium does it...
>
> Maybe even go so far as to add advanced option when creating a new wallet
> that says "this wallet allows economy fees" or not.
>
> tl;dr
> We should somehow allow for 1000 satoshi... or at least sub-10000 satoshi
> fees.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/bitpay/copay/issues/2543>.
>



-- 
BitPay.com

**Comment by tab00** [comment]
Bitcoin Wallet has the following options (in micro bitcoins):
Priority fee: 100
Normal fee: 20
Economic fee: 10

It would be good if copay has the same.

**Comment by dabura667** [comment]
Yes I agree: even during recent stress tests it was shown over 90% of transactions with at least 0.0001 BTC per kB were confirmed in the next block, so that is a good priority level.

Normal I tend to like 50 recently, as it seems to get be confirmed relatively quickly. breadwallet gives me 11-29 range and it usually takes a few blocks to confirm.

I would be up for 

Priority fee: 100
Normal fee: **50**
Economic fee: 10

But then again, idk if Copay dev team wants to have to explain this to user...

I don't use this as my day-to-day, so I don't care either way, but Windows Phone users would probably want this feature. (seeing as Copay is prolly the best)
